mkosi/__init__.py

```diff
@@ -108,7 +108,7 @@ def mount_base_trees(context: Context) -> Iterator[None]:
                 extract_tar(
                     path, d,
                     tools=context.config.tools(),
-                    sandbox=context.sandbox(options=["--bind", d, d]),
+                    sandbox=context.sandbox(options=["--ro-bind", path, path, "--bind", d.parent, d.parent]),
```

Comment:
Why do we have to bind all bases writable here?

Reply:
Because the target directory does not exist yet and is created by `tar` so we have to mount the parent directory so it can be created.